.github/workflows/changeset-snapshot/test-aperture.js

```diff
@@ -26,13 +26,14 @@ module.exports = async ({ github, context }, execa) => {
 
         try {
             // Attempt to dispatch event to PIE Aperture
-            await github.rest.actions.createWorkflowDispatch({
+            await github.rest.repos.createDispatchEvent({
```

Comment:
I can't see `ref` or `inputs` in the [docs for this method](https://octokit.github.io/rest.js/v20#:~:text=event_type%2C%0A%7D)%3B-,Parameters,-name)?

Reply:
addressed!